PHPCI/Helper/BaseCommandExecutor.php

```diff
@@ -128,6 +130,66 @@ public function executeCommand($args = array())
     }
 
     /**
+     * Manage the standard output of a process execution flushing data in
+     * realtime
+     *
+     * @param array $pipes proc_open pipes
+     * @param boolean $shouldOutput
+     */
+    protected function manageExecutionStandardOutput($process, $pipes, $shouldOutput)
+    {
+        $status = false;
+
+        if (is_resource($process)) {
+            /**
+             * don't wait to the end of the process to output stdout
+             */
+            while (!feof($pipes[1])) {
```

Comment:
are you sure that $pipes[1] is always available? Else you need to check it (maybe in the if on line 143)